### PR TITLE
Enhancement Bug Description Page: Create GitHub issue added

### PR DESCRIPTION
**Pull request by CodeWithBishal**
### Part 1
With this commit, The header widget of the issue page is matched with our final UI design
Screenshot 2024-07-07 at 3 33 43 AM

### Part 2
If the GitHub issue URL of an Issue is null, the user who created the issue and superuser will get a button to create a GitHub issue.

Upon Clicking on the button the page will hit a request to a BLT API endpoint and create a GitHub issue using GitHub REST API.

The BLT API endpoint I created has several security features, 
- No user can directly open the url - to prevent misuse of the API
- It will check if GitHub URL for that already exists before creating a new issue
- It will check if the user is logged in
- It will check if the same user who raised the issue is trying to create the GitHub issue or if it is a superuser

### Part 3
I will continue working on the design aspect of this page to match with our final design layout

### CODE REVIEW

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
A few tweaks

File: website/templates/includes/_like_dislike_widget.html [hunk 1]
```diff
@@ -167,32 +183,39 @@
...
                 <div class="flex flex-row gap-2 items-center">
                     <span class="max-sm:text-[12px]">Anonymous</span>
-                    <i class="fa-regular fa-user text-[16px]"></i>
+                    <i class="fa-regular fa-user text-[13px]"></i>
                 </div>
             </button>
         </form>
     {% endif %}
-    <button class="relative border-[1px] inline-block shadow-sm rounded-xl transform-[0] font-medium text-[#3e3446] bg-white border-black-2 text-[#0.875rem] leading-4 p-4"
+    {% comment %} edit  {% endcomment %}
```
[the remark this answers is not in the document]
Comment (CodeWithBishal) [reply website/templates/includes/_like_dislike_widget.html:197]: as in the edit button?

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
Please remove the Django comments from the html. I find them confusing. I don't think html needs comments. If you need comments then please use html comments, but id prefer no comments if you are ok with it.

File: website/templates/includes/_like_dislike_widget.html [hunk 2]
```diff
@@ -167,32 +183,38 @@
...
                     <span class="max-sm:text-[12px]">Anonymous</span>
-                    <i class="fa-regular fa-user text-[16px]"></i>
+                    <i class="fa-regular fa-user text-[13px]"></i>
                 </div>
             </button>
         </form>
     {% endif %}
-    <button class="relative border-[1px] inline-block shadow-sm rounded-xl transform-[0] font-medium text-[#3e3446] bg-white border-black-2 text-[#0.875rem] leading-4 p-4"
-            id="edit-issue">
+    {% comment %} edit  {% endcomment %}
```
[the remark this answers is not in the document]
Comment (CodeWithBishal) [reply website/templates/includes/_like_dislike_widget.html:197]: Alright Sure :)

**Review by DonnieBLT — APPROVED** [review]

**Pull request merged**